charts/gitprime-app/v2.0.0/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Might as well change this to 2.0 as well

Reply:
:sweat_smile: